### PR TITLE
Update IPTable rules and etcd template  to support persistent across reboot .    

### PR DESCRIPTION
**Pull request by g1rana**

### CODE REVIEW

File: roles/etcd/templates/etcd.j2 [hunk 1]
```diff
@@ -131,10 +131,11 @@ init_update_cluster() {
     # on master nodes, if the cluster is being initialized for first time then initialize it
     peers=""
     get_peers "{{ get_peer_addr() }}"
+    rm -rf ${ETCD_DATA_DIR}/wal/*
```
Comment (srampal) [inline roles/etcd/templates/etcd.j2]: Will this remove the etcd data directory even for the case of cluster getting updated ? That may not be what we want.

File: roles/etcd/templates/etcd.j2 [hunk 2]
```diff
@@ -131,10 +131,11 @@ init_update_cluster() {
...
     if [ "${peers}" == "" ]; then
         echo "==> no peers configured or rest of the etcd cluster is unhealthy, trying to init the cluster"
         export ETCD_INITIAL_CLUSTER_STATE=new
-        export ETCD_INITIAL_CLUSTER="
+	 export ETCD_INITIAL_CLUSTER="
```
Comment (srampal) [inline roles/etcd/templates/etcd.j2]: Minor typo, added an unnecessary extra space indentation.

Comment (g1rana) [reply roles/etcd/templates/etcd.j2]: ok

**Review by rchirakk — APPROVED** [review]
LGTM

File: roles/etcd/templates/etcd.j2 (hunk 1 above)
Comment (g1rana) [reply roles/etcd/templates/etcd.j2]: No, it won't delete data directory on update. it's only cleanup directory whenever ETCD node/container will  start or restart. During debugging it has been found that restart node/container was not able to join cluster.
This work around would let the restart ETCD node to rejoin the cluster again. Looks like this is know issue in ETCD . Found following - 
“etcd process hanging after removed from membership and restart #7512”

Comment (srampal) [reply roles/etcd/templates/etcd.j2]: ok, sounds good. 
LGTM

**Comment by g1rana** [comment]
@rchirakk @srampal 
Please review it again and merge it  if possible

**Comment by rhim** [comment]
Fixes https://github.com/contiv/netplugin/issues/963

**Pull request merged**